### ISSUE TITLE
v3.4.0 Eevee crash

### ISSUE THREAD

**Issue by nalexandru** [nalexandru/BlenderCompat#29]
Issue reported by @waltage80 on the release discussion.

[blender_debug_output.txt](https://github.com/nalexandru/BlenderCompat/files/10253639/blender_debug_output.txt)
[blender_system_info.txt](https://github.com/nalexandru/BlenderCompat/files/10253640/blender_system_info.txt)
[bug.crash.txt](https://github.com/nalexandru/BlenderCompat/files/10253641/bug.crash.txt)

**Comment by nalexandru** [comment]
@waltage80, based on your last comment from the discussion page i suspect that the cause of this crash is the "Limit Size" option "Preferences -> Viewport -> Textures" as mentioned it this bug: https://developer.blender.org/T103075 from Blender's bug tracker.

Since you have reset your preferences, can you try turning that option on and see if it causes the crash ?

**Comment by waltage80** [comment]
YES!!! this option is causes the crash of Eevee!

**Comment by nalexandru** [comment]
OK. In this case the Blender developers are aware of it, and it will probably be fixed for v3.4.1.

**Comment by waltage80** [comment]
Are you sure that BF know about this?

**Comment by nalexandru** [comment]
Yes, the link to the bug report is in the second message of this issue.

**Comment by waltage80** [comment]
3.4.1 do not solve this problem
why they release this?

**Comment by nalexandru** [comment]
I am not part of the Blender dev team. Please ask your question on their bug tracker (the report for this issue is at the link in the second comment).

**Comment by waltage80** [comment]
thank you
